packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -32,6 +32,14 @@ const propTypes = {
    * Required when using selectedPage and totalCount.
    */
   itemCountPerPage: PropTypes.number,
+  /**
+   * Allows user to set the heading id of the page.
+   */
+  paginatorHeadingId: PropTypes.string,
+  /**
+   * Allows user to set the custom paginator label.
+   */
+  paginatorLabel: PropTypes.string,
```

Comment:
Let's rename this `ariaLabel` for clarity of its usage and for consistency with Terra conventions.

Reply:
https://github.com/cerner/terra-core/pull/3867/commits/450ee57249ee65b6a8508c71973f7b2373d577ae#diff-0bceee42b89777a164b9a100804131b9447d947169e02cffe25e8791c74a958aR42